guides.md

```diff
@@ -28,6 +29,145 @@ our values.
+
+1. Sign into Mailcow: https://link.hypha.coop/email
+2. Navigate to "Mailboxes" tab
+3. Click "+Add mailbox" button at bottom of screen
+    - If the namespace is already taken by an alias, ensure it's not in use before deleting it.
+4. Fill out the following fields, noting the password:
+    - Username: info provided by new user
+    - Domain: `hypha.coop`
+    - Full name: info provided by new user
+    - Quota: `4096`
+    - Password: auto-generated
+    - Active: checked :ballot_box_with_check:
+
+We'll now log in to confirm access, and set up an initial forwarder to make first-time usage simpler.
+
+5. Open an incognito browser and access webmail login: https://mailninja.aseriesoftubez.com/SOGo/
+6. Log into the account you just created.
+7. Navigate through these menus:  
+    _:gear:(top-left) > Mail menu item > Forward tab_
+8. Check "Forward incoming mail" and enter provided primary email, clicking "Keep a copy".
+9. From your own @hypha.com email, send a welcome email, like this template:
+    >
+    > Hi XXXX --
+    >
+    > Yay! Your Hypha email's set up and forwarding to your personal inbox. People can start emailing you at this address immediately, without any extra effort on your part.
+    >
+    > Your new email (and login): XXX@hypha.coop  
+    > Your password: Sent via Signal to XXX-XXX-XXXX
+    >
+    > Having said that, when you'd like to start _sending_ email as XXX@hypha.coop, you'll need to take some more steps.
+    >
+    > The next steps will depend on how you like to handle email:
+    > - [If you'd like to use it from your existing Gmail account...]()
+    > - [If you'd like to use it from our hosted webmail interface...]()
+    > - [If you'd like to use it from another email client...]()
+    >
+    > Once you're set up, let us know by sending a test email to infra@hypha.coop, ideally with a short emoji-only story. (Keypad mashes a-ok!) We'll send a friendly poke in a week if we haven't heard back.
+    >
+    > Questions? Feedback? Reach out via infra@hypha.coop or [`#hyphacoop-infra-ops:tomesh.net`](https://chat.tomesh.net/#/room/#hyphacoop-infra-ops:tomesh.net) for chat.
+    >
+    > In Solidarity,  
+    > xxxxx
+10. Send a low-entropy Signal message with the previously noted password, for example: `🍄 password: xxxxxxxx`
+
+### Using your new inbox (Users)
+
+Accessing your inbox can be done via the hosted webmail interface or by using the same desktop email client you're already using.
+
+What you'll need:
+- Your new email address
+- Your password (sent to you by admin on initial setup)
+
+#### Using GMail
+
+1. Log into GMail
+2. Go to GMail settings, by navigating:  
+    _:gear: (top-right) > Settings > Accounts and Import_
+3. Under "Send mail as", click "Add another email address".
+    <details>
+      <summary>View Screenshot</summary>
+      <img alt="Screenshot of GMail settings page" src="https://i.imgur.com/oQMdIS9.png" />
```

Comment:
Can we keep the practice of having images in-repo?

See: https://github.com/hyphacoop/handbook/tree/master/images

Reply:
Honestly, I'd prefer not to spend time on that rn, but no block

Reply:
So I can clone the repo and use this handbook on a plane when the border opens up. But seriously, I do use these things offline a lot.

Reply:
ok, let's ok the content and i'll copy them in before merge

Reply:
agree for images in repo!